mtp_noms_ops/templates/security/credits_history_list.html

```diff
@@ -7,14 +7,25 @@
 
 
 {% block inner_content %}
+  <nav class="moj-sub-navigation" aria-label="Sub navigation">
```

Comment:
needs a `trans` wrapper

Reply:
In the `aria-label`? Happy to do so, but I worry it may wind up referring to submarines...

Reply:
🤣